### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ik-web**
[DEMO LINK](https://ik-web.github.io/react_movies-list/)

### CODE REVIEW

File: src/components/MoviesList/MoviesList.tsx [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <div className="movies">
+    {movies.map(({
+      description, imdbUrl, imgUrl, title,
+    }) => {
+      return (
+        <MovieCard
+          key={title + imdbUrl}
```
Comment (illiarerun) [inline src/components/MoviesList/MoviesList.tsx:18]: you have `imdbId` for that

File: src/components/MoviesList/MoviesList.tsx [hunk 2]
```diff
@@ -1,6 +1,27 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { Movie } from '../../types/Movie';
```
Comment (illiarerun) [inline src/components/MoviesList/MoviesList.tsx:4]: you haven't added this file to commit, please add it in next commit

File: src/components/MovieCard/MovieCard.tsx [hunk 3]
```diff
@@ -2,6 +2,47 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
```
Comment (illiarerun) [inline src/components/MovieCard/MovieCard.tsx:9]: it'd be better to pass whole object `movie` as one prop, then you can use your `Movie` type here

**Review by illiarerun — CHANGES_REQUESTED** [review]

File: src/components/MovieCard/MovieCard.tsx (hunk 3 above)
Comment (ik-web) [reply src/components/MovieCard/MovieCard.tsx]: Fixed

File: src/components/MoviesList/MoviesList.tsx (hunk 2 above)
Comment (ik-web) [reply src/components/MoviesList/MoviesList.tsx:4]: Fixed

File: src/components/MoviesList/MoviesList.tsx (hunk 1 above)
Comment (ik-web) [reply src/components/MoviesList/MoviesList.tsx]: Fixed

**Pull request closed without merging**